### PR TITLE
Lint with RuboCop Shopify

### PR DESCRIPTION
**Pull request by SeanLF**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -172,10 +193,11 @@ DEPENDENCIES
...
   rspec-sidekiq (~> 3.0.3)
+  rubocop-shopify
   shoulda-matchers (~> 4.2.0)
   sidekiq (~> 5.2.3)
   slack-notifier (~> 2.3.2)
   sqlite3 (~> 1.4.2)
 
 BUNDLED WITH
-   2.1.4
+   2.2.15
```
Comment (adrijere) [inline Gemfile.lock]: Safe?

File: rails_async_migrations.gemspec [hunk 2]
```diff
@@ -19,7 +19,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.1.4'
```
Comment (adrijere) [inline rails_async_migrations.gemspec]: I guess if the first one is safe, this one too :p

File: Gemfile.lock (hunk 1 above)
Comment (SeanLF) [reply Gemfile.lock]: Yeah that's fine, it's just bundler that was updated. I've dockerized the gem for development in #3 
The ruby image comes with bundler `2.2.15`

**Review by adrijere — APPROVED** [review]

File: rails_async_migrations.gemspec (hunk 2 above)
Comment (SeanLF) [reply rails_async_migrations.gemspec]: I'll revert the changes to the gemfile and gem spec in this PR

**Pull request merged**